### ISSUE TITLE
What could be the reason of my training accuracy decreasing

### ISSUE THREAD

**Issue by think-high** [facebookresearch/R2Plus1D#16]
Hey,

I finetuned the pretrained Sports-1m R2Plus1D model on a dataset. And below is how the training and testing accuracy looks like. (Taken from the log that the tools/train_net.py generates on completion)

![train-test accuracy](https://user-images.githubusercontent.com/5360126/41438110-e6010352-6feb-11e8-99d6-a1123db0ffc3.png)

Can you help me understand what could be the reason for the training accuracy to decrease? Logically training accuracy should never decrease with increasing epochs, right?

**Comment by patrickpoirson** [comment]
From reading the code it seems like the average training accuracy is not logged but rather the accuracy for the most recent mini batch.

**Comment by think-high** [comment]
Oh. I see. That makes sense. But won't it be a better idea to get the overall training accuracy in the log instead? @dutran

**Comment by dutran** [comment]
it is a design choice, if you choose to do a full evaluation, your training will be slow down.

**Comment by think-high** [comment]
Makes sense. Cool, Thanks.

**Comment by think-high** [comment]
@patrickpoirson, @dutran : What about the test accuracy that is written to the log file? That's for the entire testing dataset, right? 
PS: I am going to check the code to figure it out, but I just wanted to put the comment still before this issue is closed by someone. I don't want to open another issue for this. [In case I can't figure it out from the code.]

Thanks,
Rahul Bhojwani
